### PR TITLE
flake8: noqa > noqa

### PR DESCRIPTION
**Pull request by andymckay**
Apologies, I led people astray, `# flake8: noqa` ignores the entire file `# noqa` ignores just the line. In the case of a multi line import, the `# noqa` should be placed at the top. I fixed this up and an issues that resulted.

### CODE REVIEW

File: mkt/__init__.py [hunk 1]
```diff
@@ -1,18 +1,19 @@
+# flake8: noqa
```
Comment (andymckay) [inline mkt/__init__.py]: I felt that since this file was essentially dodgy import * anyway, just noqa the entire file was fine.

Comment (diox) [inline mkt/__init__.py]: Maybe add a small comment to explain we want to ignore the entire file ?

Comment (diox) [inline mkt/__init__.py]: Also, is there any hope of removing the `import *` from this file and fix all the imports that use it in zamboni ? If there is, maybe we should file a bug. No idea how big of a task it is though.

File: mkt/constants/__init__.py [hunk 2]
```diff
@@ -1,13 +1,14 @@
-from .apps import MANIFEST_CONTENT_TYPE  # flake8: noqa
+# flake8: noqa
```
Comment (diox) [inline mkt/constants/__init__.py]: Like above, a small comment would be nice.

**Comment by diox** [comment]
There are quite a few failures left to fix according to travis.

File: mkt/__init__.py (hunk 1 above)
Comment (andymckay) [inline mkt/__init__.py]: I've no idea, I'll let someone else who knows more about zamboni go for that. I hope so, its just wrong.

**Comment by andymckay** [comment]
Removed need for comments and fixed remaining failures.

**Comment by diox** [comment]
Travis hates you :) More failures to fix.

**Comment by diox** [comment]
r+

**Pull request merged**